shap/explainers/_explainer.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, model, masker, link=links.identity, algorithm="auto", output_
             the Explanation objects produced by this explainer will not have any output_names, which could effect
             downstream plots.
         """
-
+        self.name = ''
```

Comment:
might be good to still give it some name, perhaps "tabular explainer" or "selector explainer" or something along those lines?

Reply:
Sounds good.